specification/logic/resource-manager/Microsoft.Logic/stable/2016-06-01/logic.json

```diff
@@ -4921,6 +4921,232 @@
         "x-ms-odata": "#/definitions/IntegrationAccountSessionFilter"
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Logic/integrationAccounts/{integrationAccountName}/rosettanetprocessconfigurations": {
```

Comment:
>rosettanetprocessconfigurations [](start = 143, length = 31)

nit - The resource type should be camelCased for readability and to match existing convention.